src/controllers/accommodation.js

```diff
@@ -69,8 +69,8 @@ export default class AccommodationController {
    */
   static async rescindOne(req, res) {
     try {
-      const { user } = req;
-      await Accommodation.rescind(user.id);
+      const { user, query } = req;
+      await Accommodation.rescind(user.id, query.id);
       res.status(200).json({
         status: 200,
         data: 'Successfully rescinded booking for accommodation'
```

Comment:
I thought we agreed that since status property of the response object already has the status code, there should be no need including it in the json data. Hence, we should have something like this:
``` 
res.status(integer).json({
        status: string,
        ...
})
```

Reply:
Ok. I'll make the necessary changes.